ctgan/transformer.py

```diff
@@ -50,13 +50,13 @@ def _fit_discrete(self, column, data):
         ohe = OneHotEncodingTransformer()
         data = data[:, 0]
         ohe.fit(data)
-        categories = len(set(data))
+        nb_categories = ohe.transform(data).shape[1]
```

Comment:
I would rename this to `num_categories`, so it is a bit more explicit, since the `nb` could be confusing.
On the other side, since we are not really using the transformed data, if `len(ohe.dummies)` works I would use it rather than calling the transform. It's true that `transform` is not that expensive, but it is definitely more than the other option.